configuration/configuration_reference.md

```diff
@@ -10,11 +10,20 @@
                 <a href="#bases">&lt;bases/&gt;</a>
                     <a href="#base">&lt;base/&gt;</a>
             <a href="#passwordFile">&lt;passwordFile/&gt;</a>
+            <a href="#authConfigs">&lt;authConfigs&gt;</a>
```

Comment:
This is not consistent with other tags

e.g. 
```
<ldap/>
       <bases/>
             <base/>
```

While authConfig listed little differently.